news/_posts/core-weekly/2019-09-01-coreweekly-week-35-2019.md

```diff
@@ -0,0 +1,143 @@
+
+## Code changes in modules, themes & tools
+
+### Changes in developer documentation
+
+* [#330](https://github.com/PrestaShop/docs/pull/330): Add link to php cs fixer config file. Thank you [@jf-viguier](https://github.com/jf-viguier)
+
+### Contact Informations module
+
+* [#23](https://github.com/PrestaShop/ps_contactinfo/pull/23): Release v3.2.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+* [#21](https://github.com/PrestaShop/ps_contactinfo/pull/21): Re-synchronize master and dev branches, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+* [#16](https://github.com/PrestaShop/ps_contactinfo/pull/16): Version updated to v3.2.0, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+* [#15](https://github.com/PrestaShop/ps_contactinfo/pull/15): Add theme compatibility indication in configuration page, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+### Faceted search module
+
+* [#128](https://github.com/PrestaShop/ps_facetedsearch/pull/128): Release 3.2.1, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+* [#127](https://github.com/PrestaShop/ps_facetedsearch/pull/127): Bump to version 3.2.1, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+* [#126](https://github.com/PrestaShop/ps_facetedsearch/pull/126): Fix wrong nested array in maxvalue. Thank you [@matrixino](https://github.com/matrixino)
+
+* [#125](https://github.com/PrestaShop/ps_facetedsearch/pull/125): Products are displayed in wrong order when sorted by "position" and "Show products from subcategories" is disabled , by [@PierreRambaud](https://github.com/PierreRambaud)
+
+* [#124](https://github.com/PrestaShop/ps_facetedsearch/pull/124): Refactoring values sorting, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+* [#123](https://github.com/PrestaShop/ps_facetedsearch/pull/123): Better way to check for operations filters, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+### Email subscription module
+
+* [#40](https://github.com/PrestaShop/ps_emailsubscription/pull/40): Release 2.5.0, by [@Progi1984](https://github.com/Progi1984)
+
+* [#39](https://github.com/PrestaShop/ps_emailsubscription/pull/39): Release 2.5.0, by [@Progi1984](https://github.com/Progi1984)
+
+* [#37](https://github.com/PrestaShop/ps_emailsubscription/pull/37): Allow to have a custom template when rendering in a column, by [@Progi1984](https://github.com/Progi1984)
+
+### Watermark module
+
+* [#11](https://github.com/PrestaShop/watermark/pull/11): Remove "FollowSymLinks". Thank you [@antondollmaier](https://github.com/antondollmaier)
+
+<hr />
+
+Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: @jf-viguier, @PierreRambaud, @jolelievre, @Progi1984, @matthieu-rolland, @nesrineabdmouleh, @davidglezz, @boubkerbribri, @matrixino, @PeeyushAgrawalWebkul, @pojebunny, @Amazzing, @zuk3975, @202-ecommerce, @sarjon, @antondollmaier!
```

Comment:
I think the "!" brakes the link to @antondollmaier 
(last name in the list)

Reply:
Happens every time :)

Reply:
oh ok, doesn't seem to be a link actually. All good 👍 
Unless you think it's better to make it a link, your call @matks & @ttoine

Reply:
a link will be created by Jekyll when the page will be generated from the markdown.
usually, the ! is not a problem at all.
note that it is not mandatory to add a complete markdown link when using @nickname with github pages. they will be converted.